src/main/resources/ephemeral.properties

```diff
@@ -10,4 +10,5 @@ tenants=fs00000000,diku
 #######################################################
 
 fs00000000=fs00000000,{FS00000000_IU_PASSWORD}
-diku=diku,{DIKU_IU_PASSWORD}
+diku=diku_admin,{DIKU_IU_PASSWORD}
```

Comment:
Please revert this line if not needed to be changed

Reply:
reverted